gtsfm/loader/colmap_loader.py

```diff
@@ -70,10 +71,10 @@ def __init__(
         self._use_gt_extrinsics = use_gt_extrinsics
         self._max_frame_lookahead = max_frame_lookahead
 
+        # TODO in future PR: if img_fnames is None, default to using everything inside image directory
```

Comment:
Use # TODO(username) or # TODO(#issue number)

Reply:
I didn't write this originally, I just moved it.

Reply:
Oh thats worse, so we dont know who wrote this already

Reply:
Yeah I think it's pretty old